can-construct_test.js

```diff
@@ -159,3 +159,44 @@ test("setup called with original arguments", function(){
 
 	Construct.extend(o1, o2);
 });
+
+test("legacy namespace strings (A.B.C) accepted", function() {
+
+	var Type = Construct.extend("Foo.Bar.Baz");
+
+	ok(new Type() instanceof Construct, "No unexpected behavior in the prototype chain");
+	equal(Type.name, "Foo_Bar_Baz", "Name becomes underscored");
```

Comment:
Would this be different in production?  Maybe we only want to change the `eval` name, but leave the `.name` property alone?